src/problems/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from . import views
+
+app_name = 'problems'
+urlpatterns = [
+    # ex: /problems/
+    url(r'^$', views.index, name='index'),
+    # ex: /problems/5/
+    url(r'^(?P<problem_id>[0-9]+)/$', views.detail, name='detail'),
```

Comment:
I'm not sure how I'd do that right now. We can do this sometime in the future.

A related issue is how we'll organize our problems when we have a lot more, and when we introduce the concept of the "problem series".